iosxr_iso2vbox.py

```diff
@@ -401,6 +441,11 @@ def xr_cli_wait_for_output(command, pattern):
         logger.debug('Waiting 30 seconds...')
         time.sleep(30)
 
+        logger.info("Issuing shutdown command")
+        child.sendline("run shutdown -P now")
+        logger.debug('Waiting 10 seconds...')
+        time.sleep(10)
```

Comment:
It seems to be for now. Since the system shuts down, there's nothing obvious we can `expect` before continuing, which would have been preferable.